### DISCUSSION TITLE
The solver does not give a correct answer to a simple bilevel optimization problem used old input file format

### DISCUSSION THREAD

**Discussion by xlxr** [coin-or/MibS#125] [Q&A]
Hi, I have an simple example with input file written in old input file format. The formulatio is attached in picture. I also attached my input file "simple.aux.txt" and "simple.mps.txt". Note that I add an extension name ".txt" two these two files cuz otherwise Github doesn't allow me to upload. The file was running without ".txt". Also, I uploaded a file,"log.txt", which contains the output. Can anyone find out what did I do wrong? Thank you!
![formulation](https://github.com/coin-or/MibS/assets/35936942/e2fad446-b352-4e33-a2cf-099ec8c0d244)
[log.txt](https://github.com/coin-or/MibS/files/12890475/log.txt)
[simple.aux.txt](https://github.com/coin-or/MibS/files/12890495/simple.aux.txt)
[simple.mps.txt](https://github.com/coin-or/MibS/files/12890496/simple.mps.txt)

**Comment by xlxr** [comment]
MibS version 1.2 works
It can be obtained with "coinbrew fetch MibS@1.2" and built with "coinbrew build MibS@1.2"

**Reply by tkralphs** [reply]
Thanks for reporting this. There's been a lot of development recently in the `1.2` branch and we need to port this development to the `master` branch, which has not received as much attention in recent years. At the moment, `1.2` is definitely the most robust and is what should be used in most cases at the moment.

The `master` branch contains changes to `MibS` that add the ability to handle stochasticity, but it has gotten behind `1.2` in terms of ability to solve deterministic models. I am a little surprised that `master` doesn't work, though. I will look at it.
